doc/topics/domain-postmortem.md

```diff
@@ -0,0 +1,138 @@
+# domain Module Postmortem
+
+## Usability Issues
+
+### Implicit Behavior
+
+It's possible for a developer to create a new domain and then simply run
+`domain.enter()`. Which then acts as a catch-all for any exception in the
+future that couldn't be observed by the thrower. Allowing a module author to
+intercept the exceptions of unrelated code in a different module. Preventing
+the originator of the code from knowing about its own exceptions.
+
+Another is the how domains route errors automatically if no `'error'` handler
```

Comment:
Starting with "Another" sounds kind of funny here. Maybe change it to "Domains also route errors..."